.gitlab-ci/pvs/Dockerfile

```diff
@@ -15,6 +15,7 @@ RUN dnf --refresh -y upgrade --setopt=install_weak_deps=True && \
          "name=Viva64 - PVS-Studio - Stable" \
          "baseurl=https://cdn.pvs-studio.com/rpm" \
          "gpgcheck=0" \
+	 "skip_if_unavailable=True" \
```

Comment:
Indentation?

Reply:
Fixed in coming push - adding an editorconfig linter to avoid this from happening again.